tests/acceptance/browzine-primo-adapter-test.js

```diff
@@ -544,6 +544,142 @@ describe("BrowZine Primo Adapter >", function() {
+            "data": {
+              "id": 59474947,
+              "type": "articles",
+              "title": "RETRACTED ARTICLE: Flexible Field Emitter for X-ray Generation by Implanting CNTs into Nickel Foil",
+              "date": "2016-07-11",
+              "authors": "Sun, Bin; Wang, Yan; Ding, Guifu",
+              "inPress": false,
+              "doi": "10.1186/s11671-016-1523-5",
+              "ILLURL": "https://illiad.mines.edu/illiad//illiad.dll?Action=10&Form=30&&rft.genre=article&rft.aulast=Sun&rft.issn=1931-7573&rft.jtitle=Nanoscale%20Research%20Letters&rft.atitle=RETRACTED%20ARTICLE%3A%20Flexible%20Field%20Emitter%20for%20X-ray%20Generation%20by%20Implanting%20CNTs%20into%20Nickel%20Foil&rft.volume=11&rft.issue=1&rft.spage=326&rft.epage=&rft.date=2016-07-11&rfr_id=BrowZine",
+              "pmid": "27401089",
+              "openAccess": true,
+              "fullTextFile": "https://develop.libkey.io/libraries/XXXX/articles/59474947/full-text-file?utm_source=api_716",
+              "contentLocation": "https://develop.libkey.io/libraries/XXXX/articles/59474947/content-location",
+              "availableThroughBrowzine": true,
+              "startPage": "326",
+              "endPage": "",
+              "browzineWebLink": "https://develop.browzine.com/libraries/XXXX/journals/8645/issues/8065238?showArticleInContext=doi:10.1186%2Fs11671-016-1523-5&utm_source=api_716",
+              "relationships": {
+                "journal": {
+                  "data": {
+                    "type": "journals",
+                    "id": 8645
+                  }
+                }
+              },
+              "retractionNoticeUrl": "https://develop.libkey.io/libraries/XXXX/10.1186/s11671-016-1523-5"
+            },
+            "included": [
+              {
+                "id": 8645,
+                "type": "journals",
+                "title": "Nanoscale Research Letters",
+                "issn": "19317573",
+                "sjrValue": 0.782,
+                "coverImageUrl": "https://assets.thirdiron.com/images/covers/1931-7573.png",
+                "browzineEnabled": true,
+                "browzineWebLink": "https://develop.browzine.com/libraries/XXXX/journals/8645?utm_source=api_716"
+              }
+            ]
+          })
+        });
+
+        expect(request.url).toMatch(/articles\/doi\/10.1186%2Fs11671-016-1523-5/);
+        expect(request.method).toBe('GET');
+      });
+
+      afterEach(function() {
+        jasmine.Ajax.uninstall();
+        delete browzine.showFormatChoice;
+      });
+
+      it("should have an enhanced browse article in browzine option", function() {
+        var template = searchResult.find(".browzine");
+
+        expect(template).toBeDefined();
+
+        expect(template.text().trim()).toContain("View Issue Contents");
+        expect(template.text().trim()).toContain("Retracted Article");
+        expect(template.text().trim()).not.toContain("Read Article");
```

Comment:
👍 

Do you think it'd be useful to mention it should not contain the article link in the `it` statement? That seems to be the heart of this change. 

Edit: after looking at the Summon test, I see the language in the it used there

> should have an enhanced browse article showing retraction watch only

Think the same phrase could be used here, since they're both testing for only showing retraction watch?

Reply:
Good catch @karlbecker! Definitely missed that update.